### DISCUSSION TITLE
Save figures

### DISCUSSION THREAD

**Discussion by EveKorby** [MAIF/shapash#226] [Q&A]
Hiya!
Would you please guide me how I can save a figure generated in shapash? Contribution plots, importance plots, etc I mean.
It would be great if I can save them in JPEG or PNG to go in a report and would truly appreciate you advise.

Thank you
Eve

**Comment by yg79** [comment]
Hello @EveKorby!
You can save each figure.
if you use the web app: you click on the button:
<img width="421" alt="Capture d’écran 2021-07-30 à 11 09 25" src="https://user-images.githubusercontent.com/41585787/127631435-872d18d0-10ae-4cd2-90d0-3944f9ee8b2e.png">

**Reply by yg79** [reply]
You can also save using the **file_name** parameter of plot methods: 
https://shapash.readthedocs.io/en/latest/autodocs/shapash.explainer.html#the-plot-methods

**Reply by EveKorby** [reply]
Thank you @yg79 for replying, it worked perfectly :)
is there also a way to adjust the figure's dpi?

Cheers
Eve

**Reply by yg79** [reply]
No, there is no parameter for that yet, but it seems a good idea for a next release!

**Reply by yg79** [reply]
If you want to contribute Eve you are obviously welcome!

**Reply by EveKorby** [reply]
thanks @yg79 I wish I could, I am not yet that expert with Python, 
actually the good thing about Shapash was that even I could use it with my minimum background with Python and ML. 
I'll be looking forward to hear about next versions and keep on going with this amazing job! 
Eve

**Reply by thedatadecoder** [reply]
Hi @yg79 , is there any way to convert figures into html format?
